<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dell" failures="2" tests="6" name="Test runner2" time="0.423" errors="0" timestamp="2024-08-28T16:18:31 IST">
  <testcase classname="distributedparallel.test3" name="demo1" time="0.02"/>
  <testcase classname="distributedparallel.test3" name="demo2" time="0.027">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at distributedparallel.test3.demo2(test3.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase classname="distributedparallel.test2" name="demo1" time="0.04"/>
  <testcase classname="distributedparallel.test2" name="demo2" time="0.003"/>
  <testcase classname="distributedparallel.test1" name="demo1" time="0.009">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at distributedparallel.test1.demo1(test1.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- demo1 -->
  <testcase classname="distributedparallel.test1" name="demo2" time="0.009"/>
</testsuite> <!-- Test runner2 -->
